view/app-jarvis.js

```diff
@@ -58,6 +58,136 @@ app.controller('MainController', function($scope,$location,$rootScope,$http) {
 		}
 	});
 
+	$scope.getResponse = function(messageObj) {
```

Comment:
please use the function in ES6 standards. We want to maintain the uniformity throughout the entire application.

Reply:
Throughout the app functions are used in this way only.
Also we are not using any parser so the ES6 syntax might not work in some browsers.

Reply:
If thats the case then we should not support any ES6 standards. But we do need to update the js code. Also, we can do the webpack build afterwards so that the code gets compiled into old browsers. So for now, better that we go with ES6.